src/app/client/page.tsx

```diff
@@ -1,14 +1,13 @@
 import { ClientTaskCard } from '@/components/cards/ClientTaskCard'
-import { TaskRow } from '@/components/cards/TaskRow'
+// import { TaskRow } from '@/components/cards/TaskRow'
 import { Header } from '@/components/layouts/Header'
 
 export default function ClientPage() {
   return (
     <>
       <Header showCreateTaskButton={false} />
-      <TaskRow showConfigurableIcons={false} columnName="To do" taskCount={4}>
-        <ClientTaskCard />
-      </TaskRow>
+      {/* <TaskRow showConfigurableIcons={false} /> */}
```

Comment:
Remove commented code if not needed

Reply:
Client page is under construction. These commented code will get removed while working on the client page.